pages/contracts/[...catchAll].tsx

```diff
@@ -0,0 +1,15 @@
+// handle old contract paths - redirect to /explore
+import { GetServerSideProps } from "next";
+
+export default function page() {
+  return null;
+}
+
+export const getServerSideProps: GetServerSideProps = async () => {
+  return {
+    redirect: {
+      destination: "/explore",
+      permanent: false,
+    },
+  };
+};
```

Comment:
@jnsdls What do you think about this approach for redirecting old paths? 

I decided to NOT do this redirect via middleware because that would require ignoring valid routes eg: /contracts/deploy and /contracts/release - which seems like an unnecessary coupling

Reply:
I think this approach is fine